### ISSUE TITLE
An way to get next matching is needed when searching plugins while merging

### ISSUE THREAD

**Issue by MlemandPurrs** [matortheeternal/zedit#213]
In Zmerge modus, for Skyrim SE, while using 'Create Merge' in the Plugins tab where the .esp gets listed, we can use CTRL + F to search for an plugin's name, however it seems there is no method for Finding the next matching item.
Bob.esp
...tons of other .esp here...
Bob+Something Patch.esp
Searching for Bob will go to first result from top to bottom and nothing else.

**Comment by TheJeep1** [comment]
Yep....my scrolling finger is suffering.

**Comment by djserani** [comment]
Yeah, add my +1 to this. It's even worse if you can't see small text easily, since you can't make the text bigger in any way.
